browser/src/map/Map.js

```diff
@@ -186,6 +186,7 @@ L.Map = L.Evented.extend({
 				L.DomUtil.addClass(this._container.parentElement, 'readonly');
 				if (window.mode.isDesktop() || window.mode.isTablet()) {
 					L.DomUtil.addClass(L.DomUtil.get('toolbar-wrapper'), 'readonly');
+					L.DomUtil.addClass(L.DomUtil.get('toolbar-down'), 'readonly');
```

Comment:
This should go into statusbar component, update permission handler. "Similar things together".

Reply:
I don't think 
```javascript
					L.DomUtil.addClass(L.DomUtil.get('toolbar-down'), 'readonly');
```
is needed anymore :)